arbos/l1pricing/l1pricing.go

```diff
@@ -226,18 +226,22 @@ func (ps *L1PricingState) UpdateForBatchPosterSpending(statedb vm.StateDB, evm *
 	if lastUpdateTime == 0 && currentTime > 0 { // it's the first update, so there isn't a last update time
 		lastUpdateTime = updateTime - 1
 	}
-	if updateTime >= currentTime || updateTime < lastUpdateTime {
+	if updateTime > currentTime || (arbosVersion < 2 && updateTime == currentTime) || updateTime < lastUpdateTime {
 		return ErrInvalidTime
 	}
-	updateTimeDelta := updateTime - lastUpdateTime
-	timeDelta := currentTime - lastUpdateTime
+	allocationNumerator := updateTime - lastUpdateTime
+	allocationDenominator := currentTime - lastUpdateTime
+	if allocationDenominator == 0 {
+		allocationNumerator = 1
+		allocationDenominator = 1
```

Comment:
What would happen if two batches came in? Wouldn't they be trying to consume the same allocation? Could we do 1/2 instead of 1/1 as the ratio to fix this?

Reply:
Would it be better to give them an allocation of 0/1 then, to represent that no time has passed?